gui/encyclopedia/articles/ancient_world/warfare/types_of_arrowheads.json

```diff
@@ -0,0 +1,5 @@
+{
+    "title": "Types of arrowheads",
+    "subtitle": "Properties and use cases of different shapes",
+    "content": "Arrowheads came in all different sizes, forms, and functions. They are distinguished by length, width, tips, blades (lobes), barbs, sockets, and more, each type for a different purpose.\nThe general rule is: the longer and narrower a head, the more penetrating force it has. This was especially important for military arrows, as their targets were armored soldiers. Bodkin points, for example, are long square or triangular needle-like tips that excel at piercing through relatively heavy armor like leather or chainmail. Against even harder targets like medieval plate armor, though, this head was often trimmed a bit and broadened to prevent bendage.\nOn the other hand, the broader a head, the larger of a wound it can inflict (against an unarmored target). And subsequently, the more bleeding it causes and the the higher its lethality on hit. Arrowheads of hunting arrows were therefore oftentimes significantly larger than those for other purposes.\nAnother crucial attribute were barbs. Barbs are found on both military and hunting arrows, though the latter more commonly featured larger ones. While they decrease penetrating force, they make the arrow, on a hit, extremely difficult to remove without tearing open the wound. And because barbs protruded backwards, muscle movement while it's stuck in the target can cause the arrow to dig itself deeper into the flesh, doing even more damage.\nTo maximize lethality, some peoples also soaked their arrowheads in various poisonous substances before shooting.\nHowever, arrowheads also appeared in more complex shapes for very specific purposes: Unique cage-shaped arrowheads could carry incendiary charges inside. Others had small holes in them, which created a whistling noise when flying through the air and were used for sending signals. Blunt arrows, with a non-pointed tip resulting in a purposefully low penetrating power, were specifically designed for hunting small game. They would still kill their target but preserve its meat, hide, or other useful body parts.\nThere were even types of arrows for hunting ducks that were shot at a low angle and, due to special bulges on the tip, would skip along the water surface - just like flat rocks can - to the target"
```

Comment:
```suggestion
    "content": "Arrowheads came in all different sizes, forms, and functions. They are distinguished by length, width, tips, blades (lobes), barbs, sockets, and more, each type serves a different purpose.\nThe general rule is that the longer and narrower a head, the more penetrating force it has. This was especially important for military arrows, as their targets were armored soldiers. Bodkin points, for example, are long square or triangular needle-like tips that excel at piercing through relatively heavy armor like leather or chain mail. Against even harder targets like medieval plate armor, though, this head was often trimmed a bit and broadened to prevent bending.\nOn the other hand, the broader a head, the larger of a wound it can inflict (against an unarmored target). And subsequently, the more bleeding it causes and the higher its lethality on hit. Arrowheads of hunting arrows were therefore oftentimes significantly larger than those for other purposes.\nAnother crucial attribute were barbs. Barbs are found on both military and hunting arrows, though the latter more commonly feature larger ones. While they decrease the penetrating force, they make the arrow, on a hit, extremely difficult to remove without tearing open the wound. And because barbs protruded backwards, muscle movement while it's stuck in the target can cause the arrow to dig itself deeper into the flesh, doing even more damage.\nTo maximize lethality, some people also soaked their arrowheads in various poisonous substances before shooting.\nHowever, arrowheads also appeared in more complex shapes for very specific purposes: Unique cage-shaped arrowheads could carry incendiary charges inside. Others had small holes in them, which created a whistling noise when flying through the air, and were used for sending signals. Blunt arrows, with a non-pointed tip resulting in a purposefully low penetrating power, were specifically designed for hunting small game. They would still kill their target but preserve its meat, hide, or other useful body parts.\nThere were even types of arrows for hunting ducks that were shot at a low angle and, due to special bulges on the tip, would skip along the water surface - just like flat rocks can - to the target"
```